ui/lib/apps/SearchLogs/components/SearchHeader.tsx

```diff
@@ -13,40 +13,39 @@ import { useNavigate } from 'react-router-dom'
 import { useMount } from '@umijs/hooks'
 import styles from './Styles.module.css'
 import {
-  getAddress,
   namingMap,
+  NodeKind,
+  NodeKindList,
   parseClusterInfo,
   parseSearchingParams,
-  ServerType,
-  ServerTypeList,
   DATE_TIME_FORMAT,
 } from './utils'
 
 const { SHOW_CHILD } = TreeSelect
 const { RangePicker } = DatePicker
 const { Option } = Select
 
-function buildTreeData(targets: LogsearchSearchTarget[]) {
+function buildTreeData(targets: UtilsRequestTargetNode[]) {
   const servers = {
-    [ServerType.TiDB]: [],
-    [ServerType.TiKV]: [],
-    [ServerType.PD]: [],
+    [NodeKind.TiDB]: [],
+    [NodeKind.TiKV]: [],
+    [NodeKind.PD]: [],
   }
 
   targets.forEach((item) => {
-    if (item.kind === undefined) {
+    if (item === undefined || item.kind === undefined) {
       return
     }
     servers[item.kind].push(item)
   })
 
-  return ServerTypeList.filter((kind) => servers[kind].length > 0).map(
+  return NodeKindList.filter((kind) => servers[kind].length > 0).map(
     (kind) => ({
       title: namingMap[kind],
       value: kind,
       key: kind,
-      children: servers[kind].map((item: LogsearchSearchTarget) => {
-        const addr = getAddress(item)
+      children: servers[kind].map((item: UtilsRequestTargetNode) => {
+        const addr = item.display_name ?? ''
```

Comment:
can we use `!` instead? since `display_name` shall never be null

Reply:
what do you think @Deardrops

Reply:
OK